geopandas/geoseries.py

```diff
@@ -37,10 +37,10 @@ def _getitem_tuple(self, tup):
         if xs.step is not None or ys.step is not None:
             warn("Ignoring step - full interval is used.")
         xmin, ymin, xmax, ymax = obj.total_bounds
-        bbox = box(xs.start or xmin,
-                   ys.start or ymin,
-                   xs.stop or xmax,
-                   ys.stop or ymax)
+        bbox = box(xmin if xs.start is None else xs.start,
```

Comment:
yeah, the `not None` is a bit longer than `None`, but I thought it more logical to first have the passed value and if not defined the default. But I suppose you can say the same the other way around :-) 
Anyway not that important